src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSPackage.resx

```diff
@@ -168,4 +168,43 @@
   <data name="27" xml:space="preserve">
     <value>.NET Core 2015 Project Files (*.xproj);*.xproj</value>
   </data>
+  <data name="Setting_FastUpToDateCheck_Enabled_Title">
+    <value>Don't call MSBuild if a project appears to be up to date</value>
+  </data>
+  <data name="Setting_FastUpToDateCheck_LoggingLevel_Title">
+    <value>Logging Level</value>
+  </data>
+  <data name="Setting_FastUpToDateCheck_LoggingLevel_None">
+    <value>None</value>
+  </data>
+  <data name="Setting_FastUpToDateCheck_LoggingLevel_Minimal">
+    <value>Minimal</value>
+  </data>
+  <data name="Setting_FastUpToDateCheck_LoggingLevel_Info">
+    <value>Info</value>
+  </data>
+  <data name="Setting_FastUpToDateCheck_LoggingLevel_Verbose">
+    <value>Verbose</value>
+  </data>
+  <data name="Setting_FileNesting_IgnoreSolutionAndProjectSettings_Title">
+    <value>Ignore solution and project settings</value>
+  </data>
+  <data name="Setting_FileNesting_PreferSingleTargetBuilds_Title">
+    <value>Prefer single-target builds when starting a project</value>
+  </data>
+  <data name="Setting_FileNesting_PreferSingleTargetBuilds_Description">
+    <value>When starting a project that targets multiple .NET frameworks, the build time can be reduced by limiting the build to a single framework</value>
```

Comment:
Needs a trailing period.

Reply:
Also, and I know this string predates this PR, but this could be misinterpreted as referring to ".NET Framework" (big F).

How about:

> When starting a project that has multiple target frameworks, the build time can be reduced by building only a single one.

cc @tmeschter

Reply:
Let's go with that but also remove the comma in the middle:

> When starting a project that has multiple target frameworks the build time can be reduced by building only a single one.

Reply:
Changed